### ISSUE TITLE
Should the character files include their scaling?

### ISSUE THREAD

**Issue by doctornoodlearms** [genshindev/api#87]
I don't think that they should include the scaling for every level because I think it's a bit ridiculous and also unnecessary. But I do think they should include some other levels to provide an idea for how good the stats are towards late game. So I would say probably the base damage then the talent at max without constellations and then at max including constellations. And then also their stat scaling such as health, and attack?

**Comment by ArsenicBismuth** [comment]
Yeah, I also personally need info similar to the data in Genshin Fandom Wiki. There, you can also see the ascension bonus. At least with ascension bonus, it doesn't change every level (like OP suggested) and only at every 2 ascensions.
